releases/0.1x/documentation/tutorial/play.md

```diff
@@ -30,9 +32,9 @@ libraryDependencies ++= Seq(
 
 As for Play itself, this ReactiveMongo plugin requires a JVM 1.8+.
 
-The [API of this Play module](https://oss.sonatype.org/service/local/repositories/releases/archive/org/reactivemongo/play2-reactivemongo_2.12/{{site._0_1x_latest_minor}}/play2-reactivemongo_2.12-{{site._0_1x_latest_minor}}-javadoc.jar/!/index.html) can be browsed online.
+The [API of this Play module](https://oss.sonatype.org/service/local/repositories/releases/archive/org/reactivemongo/play2-reactivemongo_2.12/{{page.play_module_version}}/play2-reactivemongo_2.12-{{page.play_module_version}}-javadoc.jar/!/index.html) can be browsed online.
```

Comment:
Rather use `{{site._0_1x_latest_minor}}-play26` as elsewhere. Defining a new/page specific substitution doesn't make sense to me